docs/topics/vertx-runtime-info.adoc

```diff
@@ -1,21 +1,61 @@
 = {Vertx} Runtime Information
 
+//modularize the following sections
+.Key {Vertx} Concepts
+
+Cloud-native and container-native applications:
+
+Reactive cloud applications:
+
+.Community {Vertx} Version
+// link TBD
+Some of the {Vertx} components included in this release are not fully tested to run on OpenShift, and are provided as a link:link/to/tech/prev/table[limited-supportability Technology Preview].
+These components are based on the community version `3.4.2` of {Vertx}.
+
 .Certified {Vertx} Version
 
-{Vertx} version `3.4.2-redhat-006` is certified to run on OpenShift and provided as part of the Red Hat subscription.
+The productized {Vertx} version `3.4.2-redhat-006` is certified to run on OpenShift and provided as part of the Red Hat subscription.
 The certified {Vertx} runtime artifacts are available from the link:https://maven.repository.redhat.com/ga/[Red Hat Middleware JBoss General Availability Maven Repository].
-// link to release notes for full list of components
+//RN link TBD
+For a complete list of {Vertx} components provided as part of this release, see the link:path/to/RN#relevant_section[RHOAR Release Notes].
 
 .Installing {Vertx}
 
-To start developing applications using {Vertx}, download the BOM file containing the certified runtime artifacts available from the link:https://maven.repository.redhat.com/ga/[Red Hat Middleware JBoss General Availability Maven Repository].
+To start developing your application, download the BOM (bill of materials) file containing the certified runtime artifacts available from the link:https://maven.repository.redhat.com/ga/[Red Hat Middleware JBoss General Availability Maven Repository].
```

Comment:
You don't _download_ a Maven BOM, you just use it in your `pom.xml`. Looking below, that's exactly what you describe in "Configuring Your Application to Use Vert.x", so I'd just remove the "Installing Vert.x" section entirely.

Reply:
Good point!. I have done the same with the Upgrading section, because the steps were virtually the same for both workflows. Will update.